packages/web3torrent/src/pages/file/File.tsx

```diff
@@ -52,6 +52,13 @@ const File: React.FC<Props> = props => {
     })
   );
 
+  useEffect(() => {
+    const initResult = async () => {
+      await web3Torrent.init();
+    };
+    initResult();
+  }, [web3Torrent]);
```

Comment:
Not sure if this is the correct usage of useEffect, but I will leave it for now.

Reply:
cc @kerzhner / @geoknee / @spersico

Reply:
I think that Alex's PR supersedes this one. Might be useful to coordinate efforts to merge both pieces of code

Reply:
Good catch. This is mostly an investigation so the code here is not polished. I will focus on Alex's PR instead.